helm-chart/renku-core/templates/deployment.yaml

```diff
@@ -92,6 +92,8 @@ spec:
             {{ end }}
             - name: GIT_LFS_SKIP_SMUDGE
               value: {{ .Values.gitLFSSkipSmudge | quote }}
+            - name: RENKU_DOMAIN
+              value: {{ .Values.global.renku.domain }}
```

Comment:
Can we write down the default values for this somewhere? I don't see them in renku repository.

Also, let's please add this to `.env-example` so when we use the service with docker compose it gets picked up.

Reply:
It doesn't really have a default value, as it depends on the deployment. But it is here https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/values.yaml#L54

It's not really needed outside of a kubernetes deployment (it just defaults to `localhost` in code), so it doesn't need to be set when running with e.g. docker compose. But I can add it to the .env-example.

Reply:
Ah, okay I'm just blind then. Thanks!

Yes, please let's add it to `.env-example` and :rocket: